.github/workflows/stalebot.yaml

```
@@ -11,7 +11,7 @@ jobs:
       - uses: actions/stale@v5
         with:
           stale-issue-message: 'This issue has been automatically marked as stale due to lack of activity. You can remove the stale label or comment. Otherwise, this issue will be closed in 30 days. Thank you!'
-          close-issue-message: 'We are closing this issue due to lack of activity. Feel free to reopen it if you can provide more information. Thank you!'
+          close-issue-message: 'We are closing this issue due to lack of activity. Feel free to open a new one if you can provide more information. Thank you!'
```

There are two different approaches to this: add a comment to the bug or open a new bug. I am not entirely sure which one is preferable.

The reason for adding a comment instead of opening a new bug is that it is good to have the history around the bug even it it is closed and opening a new bug will make it harder to manage. However, then we need to have some way to notice that a new comment were added so that we can react to it. We cannot automatically open it in this case since comments can be about workarounds and other symptoms.

The reason for filing a new issue is that it is easy to add to the boards when scheduled and it will not be missed, but on the other hand we lose context of the original issue and make it harder to work with the bug.

```suggestion
          close-issue-message: 'We are closing this issue due to lack of activity. Feel free to add a comment to this issue if you can provide more information and we will re-open it. Thank you!'
```

---

I had thought about both options as well. My fear is that we might not notice that someone had responded to a closed issue. But I also see the advantages of reopening the old ticket.

---

If closed issues that have comments show up on the board, that would allow us to capture this, but it might generate a lot of extra pings. We do not know until we have tried. One option could be to add this action so that we get notifications and if that turns out to be too much, we can change strategy.

---

I am also in favor of trying this. I have changed the message accordingly.